docs/assert/true.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: true
+description: A boolean check, equivalent to Chai's assert.isTrue() and JUnit's assertTrue(). Passes if the first argument is true.
+categories:
+  - assert
+redirect_from:
+  - "/true/"
+---
+
+## `true( state [, message ] )`
+
+A boolean check, equivalent to Chai's assert.true() and JUnit's assertTrue(). Passes if the first argument is `true`.
+
+| name               | description                          |
+|--------------------|--------------------------------------|
+| `state`            | Expression being tested              |
```

Comment:
This is a left-over from `ok()`, for which in turn I actually don't know where it originates, but in general we call this the `actual` value. I'll use that here as well when landing the commit.